static/lang/re-en.json

```diff
@@ -5527,6 +5527,10 @@
                 "Sight": "Low-light vision and darkvision",
                 "Speed": "+20-foot status bonus to Speed"
             },
+            "SpellEffectFieryBody": {
+                "ReturnDamageNote": "Return @Damage[3d6[fire]] damage to attacker.",
+                "ReturnDamageHeightenedNote": "Return @Damage[4d6[fire]] damage to attacker."
+            },
```

Comment:
```suggestion
            "SpellEffectFieryBody": {
                "Note": "The attacker takes @Damage[(ternary(gte(@item.level,9),4,3))d6[fire]] damage."
            },
```

Reply:
Reason why I went with hardcoded value is to reuse it in Furnace Form, which does not have a heightened variant, but can be lvl 9 and more, could this be achieved without copying the string?

Reply:
Furnace form is its own effect already, and even if it grants this effect the level will just be the base level of the furnace form feat. The heightening only happens when casting the spell so wouldn't matter.